package.json

```diff
@@ -13,6 +13,7 @@
   "type": "module",
   "scripts": {
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test:integration": "node --experimental-vm-modules node_modules/jest/bin/jest.js test/integration",
```

Comment:
Might need to just used the test script due to the way the FFC pipelines are configured in Jenkins.